scripts/codemod.mjs

```diff
@@ -1,22 +1,17 @@
 // @ts-check
 import * as Glob from "glob"
-import jscodeshift from "jscodeshift/src/Runner.js"
-import * as Fs from "node:fs"
+import * as Jscodeshift from "jscodeshift/src/Runner.js"
 import * as Path from "node:path"
 
-const packageJsonPath = Path.resolve("package.json")
-const packageJson = JSON.parse(Fs.readFileSync(packageJsonPath, "utf-8"))
-const workspaces = Glob.globSync(packageJson["workspaces"])
-const packages = workspaces.map((workspace) => workspace.replace("packages/", ""))
-const pattern = `packages/{${packages.join(",")}}/src/**/*.ts`
-
+// Look up files in all workspace packages including those nested in sub-packages (e.g. `packages/ai/openapi`).
+const pattern = "packages/{*,*/*}/src/**/*.ts"
```

Comment:
We have other places in our config where the nested dirs `packages/ai/*` are not considered properly.

Reply:
This is somewhat burdensome to maintain because we have to keep that in mind everytime we add a new subdir package group. Instead, we should probably just simplify this as I did here. It's unlikely that this will cause false-positives.